app/models/program.rb

```diff
@@ -15,4 +15,5 @@ class Program < ActiveRecord::Base
   }.freeze
 
   validates :facility_code, presence: true
+  belongs_to :institution
```

Comment:
This line should probably just be deleted.
`programs` doesn't have a `institution_id` field.
It does have a `facility_code`, but it doesn't have a `version` attribute.  That is, there will be multiple `institutions` records (one per version) for each `program`

Reply:
Got it. Changes have been submitted.